### PR TITLE
A gerrit service.

### PR DESCRIPTION
**Pull request by ralphbean**
It works!

I noticed while doing this that the IssueService.validate_config call is still
non-idiomatic, fwiw.

### CODE REVIEW

**Comment by ryneeverett** [comment]
Did you intend to commit gerrit.py?

File: bugwarrior/services/gerrit.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            if not config.has_option(target, option):
+                die("[%s] has no '%s'" % (target, option))
+
+        IssueService.validate_config(config, target)
+
+    def issues(self):
+        url = self.url + '/a/changes/' + \
+            '?q=is:open+is:reviewer' + \
+            '&o=MESSAGES&o=DETAILED_ACCOUNTS'
+        response = self.session.get(url)
```
Comment (ryneeverett) [inline bugwarrior/services/gerrit.py]: Why not:

```py
response = self.session.get(
    self.url,
    {'q': 'is:open+is:reviewer', 'o': ['MESSAGES', 'DETAILED_ACCOUNTS']})
```

File: bugwarrior/services/gerrit.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        changes = json.loads(response.text[4:])
```
Comment (ryneeverett) [inline bugwarrior/services/gerrit.py]: What are you slicing out of the response here?

**Comment by ralphbean** [comment]
> Did you intend to commit gerrit.py?

Aww... no.  :(  Time to change that pw.

File: bugwarrior/services/gerrit.py (hunk 2 above)
Comment (ralphbean) [inline bugwarrior/services/gerrit.py]: It's some ``)]}'`` garbage.  Will add a comment.

File: bugwarrior/services/gerrit.py (hunk 1 above)
Comment (ralphbean) [inline bugwarrior/services/gerrit.py]: Ah, because requests will percent-encode the ``:`` characters, and gerrit doesn't like that.  :(

**Comment by ralphbean** [comment]
@ryneeverett, updated and rebased.

**Comment by ryneeverett** [comment]
LGTM

**Pull request merged**